config/tf_modules/gcp-k8s-service/variables.tf

```
@@ -12,6 +12,7 @@ locals {
   env_short       = substr(var.env_name, 0, 9)
   layer_short     = substr(var.layer_name, 0, 9)
   module_short    = substr(var.module_name, 0, 9)
+  get_buckets     = concat(var.read_buckets, var.write_buckets)
```

```suggestion
  get_buckets     = toset(concat(var.read_buckets, var.write_buckets))
```

`toset` removes any duplicate values